src/Core/State/LegacyDataPersisterProcessor.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\State;
+
+use ApiPlatform\Core\DataPersister\DataPersisterInterface;
+use ApiPlatform\Core\DataPersister\ResumableDataPersisterInterface;
+
+class LegacyDataPersisterProcessor implements ProcessorInterface
+{
+    public function __construct(private DataPersisterInterface $dataPersister)
+    {
+    }
+
+    public function resumable(array $context = []): bool
+    {
+        if ($this->dataPersister instanceof ResumableDataPersisterInterface) {
+            return $this->dataPersister->resumable($context);
+        }
+
+        return false;
+    }
+
+    public function supports($data, array $identifiers = [], array $context = []): bool
+    {
+        if ($this->dataPersister instanceof ResumableDataPersisterInterface) {
+            return $this->dataPersister->supports($context);
+        }
+
+        return false;
+    }
+
+    public function process($data, array $identifiers = [], array $context = [])
+    {
+        dd('passons');
```

Comment:
ici il va falloir identifier:
- soit que l'on supprime une resource (appeler $this->dataPersister->remove)
- soit que l'on veut créer ou modifier (appeler $this->dataPersister->persist)

Ensuite, comme pour le REadListener il faut ajouter le Processor dans le WriteListener,